include/bluetooth/bluetooth.h

```diff
@@ -1215,6 +1215,41 @@ int bt_le_per_adv_sync_recv_enable(struct bt_le_per_adv_sync *per_adv_sync);
  */
 int bt_le_per_adv_sync_recv_disable(struct bt_le_per_adv_sync *per_adv_sync);
 
+/**
+ * @brief Add a device to the periodic advertising list.
+ *
+ * Add peer device LE address to the periodic advertising list. This will make
+ * it possibly to automatically create a periodic advertising sync to this
+ * device.
+ *
+ * @param addr Bluetooth LE identity address.
+ * @param sid  The advertising set ID.
```

Comment:
Where does the API user get this value? Is there a valid range for it? I think it would be helpful to document this.

Reply:
Good idea to add that. Extended the documentation stating where the value can be obtained from.